Trex Runner

RAHULBAWA777 opened the pull request:
# Description

added Trex runner game with additional functionalities

Fixes: #617

<!---give the issue number you fixed----->

## Type of change

added new project

- [x ] New feature (non-breaking change which adds functionality)


# Checklist:

<!----Please delete options that are not relevant.And in order to tick the check box just but x inside them for example [x] like this----->
- [ x] I have made this from my own
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings


# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
https://rahulbawa777.github.io/T-Rex-Runner-game-/

DEEPALI-SUK reviewed (changes requested):
@RAHULBAWA777 make it responsive and add in style_links.json and add preview.png

RAHULBAWA777 commented:
hey i have updated style_link.json
 also added preview.png 
also, the is responsive .... I have not made it extremely responsive to mobile devices because the game canvas will become ultra-small to play ... so it is best as it is

DEEPALI-SUK reviewed (changes requested):
Instaed of creating new pr, do changes here only it will be reflected in this pr itself

DEEPALI-SUK commented:
> Instaed of creating new pr, do changes here only it will be reflected in this pr itself

Instead*

DEEPALI-SUK commented:
https://deploy-preview-627--amazing-js-projects.netlify.app/t-rex-runner-game-/

RAHULBAWA777 commented:
> Instaed of creating new pr, do changes here only it will be reflected in this pr itself

hey let me tell you I am a beginner in git and github...so I have no idea how to do that ....can you please guide me !!

DEEPALI-SUK commented:
> > Instaed of creating new pr, do changes here only it will be reflected in this pr itself
> 
> hey let me tell you I am a beginner in git and github...so I have no idea how to do that ....can you please guide me !!

1. pull the changes 
2. do changes in this branch only and then add and commit and push, changes will be reflected here

RAHULBAWA777 commented:
> > > Instaed of creating new pr, do changes here only it will be reflected in this pr itself
> > 
> > 
> > hey let me tell you I am a beginner in git and github...so I have no idea how to do that ....can you please guide me !!
> 
> 1. pull the changes
> 2. do changes in this branch only and then add and commit and push, changes will be reflected here

I have added this trex project through github uploads by dragging and dropping it to the repo directly ...i have not used git .

DEEPALI-SUK commented:
> > > > Instaed of creating new pr, do changes here only it will be reflected in this pr itself
> > > 
> > > 
> > > hey let me tell you I am a beginner in git and github...so I have no idea how to do that ....can you please guide me !!
> > 
> > 
> > 
> > 1. pull the changes
> > 2. do changes in this branch only and then add and commit and push, changes will be reflected here
> 
> I have added this trex project through github uploads by dragging and dropping it to the repo directly ...i have not used git .

you can watch some youtube tutorials that use git (this will be helpful if u have to do opensource) or
close this pr and create new pr with all changes 

I would suggest u to watch some basic tutorials on git
